### PR TITLE
SW-1580 Clean up Add buttons

### PR DESCRIPTION
**Pull request by alexTerraformation**
Add plus icon to add buttons on desktop.  Make all add buttons (except for empty state) only plus icon on mobile.
Screen Shot 2022-10-04 at 8 34 52 PM
Screen Shot 2022-10-04 at 8 35 05 PM
Screen Shot 2022-10-04 at 8 35 23 PM
Screen Shot 2022-10-04 at 8 36 03 PM
Screen Shot 2022-10-04 at 8 36 16 PM
Screen Shot 2022-10-04 at 8 36 39 PM
Screen Shot 2022-10-04 at 8 36 44 PM
Screen Shot 2022-10-04 at 8 37 20 PM
Screen Shot 2022-10-04 at 8 37 28 PM
Screen Shot 2022-10-04 at 8 38 03 PM
Screen Shot 2022-10-04 at 8 38 37 PM
Screen Shot 2022-10-04 at 8 43 31 PM

### CODE REVIEW

**Comment by rachel-tf-design** [comment]
Looks great! Thanks @alexTerraformation :)

**Review by constanzauanini — APPROVED** [review]

**Comment by charlie-tf** [comment]
image
curious about the mobile Viability Testing Add button. Can it be made a circular blue add button like the other mobile ones? It is just sliiightly wider than it is tall 🙂

File: src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx [hunk 1]
```diff
@@ -43,7 +43,8 @@ export default function ViabilityTestingPanel(props: ViabilityTestingPanelProps)
           <Box sx={{ marginTop: 4 }}>
             <Button
               priority='secondary'
-              label={strings.START_TESTING}
+              label={strings.ADD_TEST}
```
[the remark this answers is not in the document]
Comment (alexTerraformation) [reply src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx:46]: Yes, this was requested by the design team

**Comment by alexTerraformation** [comment]
> Can it be made a circular blue add button like the other mobile ones? It is just sliiightly wider than it is tall

We can change it, but even the one that is circular is more coincidence than not.  It just so happens that the plus icon in that size of button with that amount of padding === circle.  Other icons or other button sizes lead to non-circular shapes.  I talked about this with @rachel-tf-design who agreed that for now non-circular is fine and we may revisit in the future.

**Pull request merged**